Bottom: 49b405a11672fcbef9ae0c0ecd222f3334912941
Top:    eb7a9ad0d044f0452e9fd12d745f5dc7fc07dcb1
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-11 09:55:52 +0800

Refresh of autofs-5.0.7-make-dump-maps-check-for-duplicate-indirect-mounts.patch

---

diff --git a/lib/master.c b/lib/master.c
index f2a9c0d..6143a9c 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1352,7 +1352,8 @@ int master_show_mounts(struct master *master)
 
 			if (count && ap->type == LKP_INDIRECT)
 				printf("\n  duplicate indirect map source for"
-					" %s, will be ignored at run time\n");
+					" %s, will be ignored at run time\n",
+					ap->path);
 
 			if (source->type)
 				printf("\n  type: %s\n", source->type);
